WordPress/Classes/ViewRelated/Views/WPAnimatedBox.m

```diff
@@ -73,8 +73,6 @@ - (void)prepareAnimation:(BOOL)animated
                              view.transform = CGAffineTransformMakeTranslation(0, CGRectGetHeight(self.frame) - YOrigin);
                          }
                      }];
-
-    _isPreparedToAnimate = YES;
```

Comment:
No.  The position of this call was wrong, but you're right in that this property is missing its initialization.  Good catch.